Dockerfile

```diff
@@ -1,5 +1,5 @@
 ARG REG=100225593120.dkr.ecr.us-east-1.amazonaws.com
-ARG DOCKER_PULL_TAG=latest
+ARG DOCKER_PULL_TAG=stage
```

Comment:
Same thing here, if possible leave `latest` as default and define a different variable value at execution rather than changing the default.

Reply:
In vein of the comment I made [here](https://github.com/alliance-genome/agr_file_generator/pull/148#discussion_r1300240795), using DOCKER_PULL_TAG=stage might be the correct thing here. Resolving this conversation.